### PR TITLE
Blockquote support

### PR DESCRIPTION
**Pull request by patsy-issa**
Within a post's excerpt:

post-excerpt

Post body:

post-body

post-body-2

### CODE REVIEW

File: _posts/2017-02-13-npm-libs-in-ember-cli.md [hunk 1]
```diff
@@ -161,7 +161,12 @@ make sure that Ember CLI waits for the installation to finish.
 
 ## npm vs. Bower
 
-<blockquote class="tweet" data-lang="de"><p lang="en" dir="ltr">&quot;What&#39;s bower?&quot;<br>&quot;A package manager, install it with npm.&quot;<br>&quot;What&#39;s npm?&quot;<br>&quot;A package manager, you can install it with brew&quot;<br>&quot;What&#39;s brew?&quot;<br>...</p>&mdash; Stefan Baumgartner (@ddprrt) <a href="https://twitter.com/ddprrt/status/529909875347030016">5. November 2014</a></blockquote>
+> &quot;What&#39;s bower?&quot;  
+> &quot;A package manager, install it with npm.&quot;  
+> &quot;What&#39;s npm?&quot;  
+> &quot;A package manager, you can install it with brew&quot;  
+> &quot;What&#39;s brew?&quot; ...  
+> <author>&mdash; Stefan Baumgartner (@ddprrt) <a href="https://twitter.com/ddprrt/status/529909875347030016">5. November 2014</a></author>
```
Comment (marcoow) [inline _posts/2017-02-13-npm-libs-in-ember-cli.md]: Can we add the `&mdash;` via CSS?

**Review by marcoow — COMMENTED** [review]
Cool 👍 I think it'd be good for @marhigh to have a look as well

**Comment by marhigh** [comment]
Is there any way to guarantee that there are always quotes around the quote? They are missing in the "ember-cli-deploy-notifications" post excerpt.

**Comment by mansona** [comment]
@marhigh we could add them via CSS 😂  we've recently done that in the new ember-blog redesign work https://deploy-preview-7--ember-blog-legacy.netlify.app/the-emberjs-times-issue-49/#initialexperiments (although it's a bit more stylised)

**Comment by marhigh** [comment]
> @marhigh we could add them via CSS 😂 we've recently done that in the new ember-blog redesign work https://deploy-preview-7--ember-blog-legacy.netlify.app/the-emberjs-times-issue-49/#initialexperiments (although it's a bit more stylised)

I will be working on a more stylized version but I don't think it should block this. From my perspective, we can merge this for now and make sure they are consistently displaying the quotes or a quote icon when we style them further.

**Comment by marhigh** [comment]
I just added the stylized version, you can take a look at the design & spec here: https://share.goabstract.com/8ae826bc-280d-4975-b530-b280ecf12afd

This task will also resolve https://github.com/simplabs/simplabs.github.io/issues/1020

**Comment by patsy-issa** [comment]
Et voila, we have pretty quotes now 😍 

Screenshot 2020-09-04 at 17 16 15
Screenshot 2020-09-04 at 17 18 07
Screenshot 2020-09-04 at 17 18 38

File: _posts/2015-12-3-ember-cli-deploy-notifications.md [hunk 2]
```diff
@@ -13,7 +13,14 @@ topic: ember
...
 
-<blockquote class="tweet" lang="de"><p lang="en" dir="ltr">ember-cli-deploy 0.5 is now released and ready for use with a great docs site and already-rich plugin ecosystem: <a href="https://t.co/6yhjmjQrYD">https://t.co/6yhjmjQrYD</a></p>&mdash; Luke Melia (@lukemelia) <a href="https://twitter.com/lukemelia/status/659787938625134592">29. Oktober 2015</a></blockquote> <script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+> ember-cli-deploy 0.5 is now released and ready for use with a great docs site
+> and already-rich plugin ecosystem:
+> <a href="https://t.co/6yhjmjQrYD">https://t.co/6yhjmjQrYD</a> > <author>Luke
+> Melia (@lukemelia)
+> <a href="https://twitter.com/lukemelia/status/659787938625134592">29. Oktober
+> 2015</a></author>
+>
+> <script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
```
Comment (marcoow) [inline _posts/2015-12-3-ember-cli-deploy-notifications.md]: is this still going to work as intended when the `blockquote` doesn't have the `tweet` class anymore?

File: _posts/2015-12-3-ember-cli-deploy-notifications.md [hunk 3]
```diff
@@ -13,7 +13,14 @@ topic: ember
 A few weeks ago a new version of the _"official"_ ember deployment solution
 [ember-cli-deploy](http://ember-cli-deploy.com/) was released:
 
-<blockquote class="tweet" lang="de"><p lang="en" dir="ltr">ember-cli-deploy 0.5 is now released and ready for use with a great docs site and already-rich plugin ecosystem: <a href="https://t.co/6yhjmjQrYD">https://t.co/6yhjmjQrYD</a></p>&mdash; Luke Melia (@lukemelia) <a href="https://twitter.com/lukemelia/status/659787938625134592">29. Oktober 2015</a></blockquote> <script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+> ember-cli-deploy 0.5 is now released and ready for use with a great docs site
+> and already-rich plugin ecosystem:
+> <a href="https://t.co/6yhjmjQrYD">https://t.co/6yhjmjQrYD</a> > <author>Luke
```
Comment (marcoow) [inline _posts/2015-12-3-ember-cli-deploy-notifications.md]:
```suggestion
> <a href="https://t.co/6yhjmjQrYD">https://t.co/6yhjmjQrYD</a>
> <author>Luke
```

**Comment by marhigh** [comment]
Sorry about the very slow review! 

There are a few things that we should update: 
1. [In blog post] The font size is not responsive (should be 18px on mobile) 
2. The border should end at the baseline of the quote's author. The border's color should be updated.
3. Double-check spacing, currently there is 10px between quote + author (should be 16px). 
4. Revisit font sizes / styling of the quotes in the blog preview (from Abstract: "The font size of the quote always matches the body font size that surrounds it, and its responsive behavior also matches that.")

Screen Shot 2020-09-14 at 18 31 57

File: _posts/2015-12-3-ember-cli-deploy-notifications.md (hunk 2 above)
Comment (patsy-issa) [reply _posts/2015-12-3-ember-cli-deploy-notifications.md]: I am not entirely sure it was working in the first place 👀 

Not seeing much of a difference between [production](https://simplabs.com/blog/2015/12/3/ember-cli-deploy-notifications/) and the [preview](https://deploy-preview-1260--objective-northcutt-37494c.netlify.app/blog/2015/12/3/ember-cli-deploy-notifications/), not too familiar with twitter but what is this widget supposed to do?

Comment (marcoow) [reply _posts/2015-12-3-ember-cli-deploy-notifications.md]: Hm, right – hasn't been working for some time then apparently. I think we can just remove it altogether then and somehow make it clear this was a tweet? What do you think would be a good way to do that @marhigh ?

Comment (marhigh) [reply _posts/2015-12-3-ember-cli-deploy-notifications.md]: Sorry, just saw this. Agreed - we can create a new issue for a stylized tweet and deal with that separately.

**Review by marcoow — APPROVED** [review]

**Comment by marhigh** [comment]
Hey Patsy! I still see some changes needed: 

Screen Shot 2020-10-09 at 17 51 40
(Screenshot at 1920px viewport width)
This font size (& quote size) still seems to be mismatched - please take a look at the last point in the description: 
"The font size of the quote always matches the body font size that surrounds it, and its responsive behavior also matches that. The square part of the quotation mark matches the x-height of the body font size that surrounds it, and is aligned to the baseline of the first line of text." 

https://share.goabstract.com/37b460bf-0b64-46be-bc12-6149801017c4


Screen Shot 2020-10-09 at 17 55 54
In the blog post lead, the same font size (& quote size) mismatch. This one is not using the space system for top and bottom margins (I see 30px instead of 32px).
Border uses #5b5b5b instead of #757575.

Happy to go over these small details together in a call. There might be a few edge cases that are not currently accounted for in the design (e.g. loud quote in lead of blog post)

**Comment by marcoow** [comment]
@marhigh: please check again – I think this should now be fine

**Comment by marhigh** [comment]
@marcoow I don't see the changes applied from this last feedback (all items are still visible on Michael's post "ember-cli-deploy-notifications"). Could you please double check?

Aside from that, how shall I test the quote in the very first blog preview since it's larger than the rest? 


> Hey Patsy! I still see some changes needed:
> 
> Screen Shot 2020-10-09 at 17 51 40
> 
> (Screenshot at 1920px viewport width) This font size (& quote size) still seems to be mismatched - please take a look at the last point in the description: "The font size of the quote always matches the body font size that surrounds it, and its responsive behavior also matches that. The square part of the quotation mark matches the x-height of the body font size that surrounds it, and is aligned to the baseline of the first line of text."
> https://share.goabstract.com/37b460bf-0b64-46be-bc12-6149801017c4
> 
> Screen Shot 2020-10-09 at 17 55 54
> 
> In the blog post lead, the same font size (& quote size) mismatch. This one is not using the space system for top and bottom margins (I see 30px instead of 32px). Border uses #5b5b5b instead of #757575.
> Happy to go over these small details together in a call. There might be a few edge cases that are not currently accounted for in the design (e.g. loud quote in lead of blog post)

**Comment by marcoow** [comment]
@marhigh I'm not sure which elements you mean that are still there (but shouldn't)? My understanding was the font sizes were wrong and that should be fixed now? Did I miss a place where they are still wrong maybe?

**Comment by marhigh** [comment]
@marcoow yes, both the places that I snapshot are still the same (so my comments on the previous review are all still open)

Michael's blog post preview: https://deploy-preview-1260--objective-northcutt-37494c.netlify.app/blog/page/5
Michael's blog post: https://deploy-preview-1260--objective-northcutt-37494c.netlify.app/blog/2015/12/3/ember-cli-deploy-notifications/

Furthermore:
- Border left still #5b5b5b instead of #757575
- Quote icon is also #5b5b5b instead of #757575

And finally, I don't know how to test for a quote in the very first blog preview. Could you advise? It is larger than the rest of the previews.

**Comment by marcoow** [comment]
Hm, here https://deploy-preview-1260--objective-northcutt-37494c.netlify.app/blog/2015/12/3/ember-cli-deploy-notifications/ we're using the same style as for the rest of the lead text? Was that not the intention – a quote that appears in the lead of a post to have the same size as the rest of the lead text?

I'll fix the size on the overview page 👍 

Just to be sure there's no mis-understanding – #757575 isn't used anywhere else so introducing a new color is the idea here?

**Comment by marhigh** [comment]
For: https://deploy-preview-1260--objective-northcutt-37494c.netlify.app/blog/2015/12/3/ember-cli-deploy-notifications/ 
The size of the quote icon does not match the font size. The font size is already correct. 

I had #757575 in the design file, clearly an outdated value. Since that is not already a color variable then we should not introduce it. Disregard the update of color 👍

**Comment by marcoow** [comment]
I fixed the font size on the overview page. I don't think I know how to do

> The square part of the quotation mark matches the x-height of the body font size that surrounds it, and is aligned to the baseline of the first line of text.

Maybe @pichfl and I can look at that on thursday.

**Comment by marcoow** [comment]
@marhigh can we merge this without fixing

> The square part of the quotation mark matches the x-height of the body font size that surrounds it, and is aligned to the baseline of the first line of text.

I'm not sure how to do that and it seems like we'd spend a potentially large amount of time on something very minor?

**Comment by marhigh** [comment]
> @marhigh can we merge this without fixing
> 
> > The square part of the quotation mark matches the x-height of the body font size that surrounds it, and is aligned to the baseline of the first line of text.
> 
> I'm not sure how to do that and it seems like we'd spend a potentially large amount of time on something very minor?

As you wish.

**Pull request merged**